### PR TITLE
[MTDB-11629] Fix/info button

### PR DESCRIPTION
**Pull request by catboop**
still need to finalize instructions and probably refine styling, but here's a start lol

Screenshot 2023-11-02 at 2 59 53 PM

### CODE REVIEW

File: src/components/WelcomeModal/Content.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+}
+
+const Content: FC<Props> = ({ returningUser }) => {
+  const { close } = useModal();
+
+  return (
+    <>
+      <Moon src={verseMoon} />
+      {returningUser ? <ReturningUser /> : <NewUser />}
+      <StyledButton onClick={close}>Play</StyledButton>
```
Comment (micahpatng) [inline src/components/WelcomeModal/Content.tsx:20]: you have to pass `modalRef` too. currently clicking `Play` doesn't close the modal

**Comment by PorlyBe** [comment]
swap step 2 and 3 but love  the instructions

File: src/components/WelcomeModal/NewUser.tsx [hunk 2]
```diff
@@ -9,6 +9,23 @@ const NewUser = () => {
         Click for verse, climb the leaderboard! Join the Verse community and
         experience a world of endless clicking fun.
       </Description>
+      <Description style={{ textAlign: 'left' }}>
+        Here is how to play:
+        <br />
+        <br />
+        1. Click the Verse moon to earn points
```
Comment (micahpatng) [inline src/components/WelcomeModal/NewUser.tsx:16]: I think @PorlyBe wanted to include the verse moon icon here so users can easily distinguish which one is it

Comment (PorlyBe) [reply src/components/WelcomeModal/NewUser.tsx:16]: i did but wasn't thinking there's a big moon image at the top of that modal already😄

File: src/components/WelcomeModal/Content.tsx (hunk 1 above)
Comment (catboop) [reply src/components/WelcomeModal/Content.tsx:19]: ah whoops, great catch. thank you @micahpatng !

**Review by micahpatng — APPROVED** [review]

**Comment by allisonle** [comment]
@catboop Looks like we've settled on using 'Tools' instead of 'Buildings', so copy here should be updated

**Pull request merged**